ReactNativeClient/lib/models/Setting.js

```diff
@@ -387,6 +387,7 @@ class Setting extends BaseModel {
 			'markdown.plugin.emoji': { value: false, type: Setting.TYPE_BOOL, section: 'plugins', public: true, appTypes: ['mobile', 'desktop'], label: () => `${_('Enable markdown emoji')}${wysiwygNo}` },
 			'markdown.plugin.insert': { value: false, type: Setting.TYPE_BOOL, section: 'plugins', public: true, appTypes: ['mobile', 'desktop'], label: () => `${_('Enable ++insert++ syntax')}${wysiwygNo}` },
 			'markdown.plugin.multitable': { value: false, type: Setting.TYPE_BOOL, section: 'plugins', public: true, appTypes: ['mobile', 'desktop'], label: () => `${_('Enable multimarkdown table extension')}${wysiwygNo}` },
+			'markdown.plugin.video': { value: false, type: Setting.TYPE_BOOL, section: 'plugins', public: true, appTypes: ['desktop'], label: () => `${_('Enable video embedding')}${wysiwygNo}` },
```

Comment:
This works on mobile too as far as I can see so please enable it in this case too.

Reply:
okay